zebra-consensus/src/transaction/tests.rs

```diff
@@ -266,6 +269,7 @@ async fn v5_transaction_is_rejected_before_nu5_activation() {
                 height: canopy
                     .activation_height(network)
                     .expect("Canopy activation height is specified"),
+                time: chrono::MIN_DATETIME,
```

Comment:
Likely Test Bug: This change creates extra test failures. If the block time is the minimum block time, there is no valid unlock time, because the unlock time must be less than the block time.

```suggestion
                time: chrono::MAX_DATETIME,
```

Reply:
Done in [e0ad6e4](https://github.com/jvff/zebra/commit/e0ad6e4ce03afd998ae5703eeab39f72d2e18ec4).